bin/lint_all.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+script_name=$( basename "$0" )
+script_version="0.0.2"
+usage="Usage: ${script_name} [-v|--version|-h|--help]"
+
+# Parse the command line parameters
+while [[ $# -gt 0 ]]
+do
+    param="$1"
+    case "${param}" in
+        -v|--version)
+            echo "${script_name}" version ${script_version}
+            echo "Copyright (C) xarxziux 2018-9"
```

Comment:
I don't mind giving you attribution, but can't we do a header comment (with your username) with license? This currently is not indicative of the license and discourages modification / extension?

Reply:
This was copy-pasted from another project and doesn't belong here.  I've removed the line.